helper-cli/src/main/kotlin/commands/SaveResultWithResolvedScopesCommand.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
```

Comment:
Should be 2022.

Reply:
Actually, I wrote this in 2021, but fixed :)